problem-26/src/app/component/todo-list/todo-list.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component, DestroyRef, inject, OnInit} from '@angular/core';
+import {NgForOf} from "@angular/common";
+import {TodoService} from "../../service/todo.service";
+import {Todo} from "../../model/todo.modal";
+import {takeUntilDestroyed} from "@angular/core/rxjs-interop";
+import {map} from "rxjs";
+
+@Component({
+  selector: 'app-todo-list',
+  standalone: true,
+  imports: [
+    NgForOf
+  ],
+  template:
+    `
+    <div class="main-container">
+      <h3>List of ToDo's!</h3>
+      <div *ngFor="let todo of todoList " class="todo-card">
+            <p class="todo-title">
+              {{todo.title}}
+            </p>
+      </div>
+
+    </div>
+  `
+  ,  styleUrl: './todo-list.component.scss'
+})
+export class TodoListComponent implements OnInit{
+  private todoService: TodoService = inject(TodoService);
+  private destroyRef: DestroyRef = inject(DestroyRef);
+
+  public todoList: Todo[] = [];
+
+  ngOnInit(): void {
```

Comment:
This is not a real-world example at all.

Try to find a new exercise for RxJS operators.

Hint: Resolve nested subscriptions with the RxJS switchMap operator.

Reply:
Resolved - replaced with question that uses RxJS switchMap operator.